package.json

```diff
@@ -7,8 +7,8 @@
     "dev": "ng serve --host 0.0.0.0 --disable-host-check",
     "build": "ng build",
     "watch": "ng build --watch --configuration development",
-    "test": "ng test --code-coverage",
-    "test:watch": "ng test --watch --code-coverage",
+    "test": "ng test --code-coverage --browsers=ChromeHeadless",
```

Comment:
--browsers=ChromeHeadless will have the unit testing run headless, means there won't be any browser popped up to show whatever being initialized/tested by Jasmine/Karma.